src/js/createCardItem.js

```diff
@@ -0,0 +1,31 @@
+const cardContainer = document.querySelector('.grid-cards-container');
+
+const createCardItem = (item, itemLike = { likes: 0 }) => {
+  const cardElement = document.createElement('article');
+  const cardImage = document.createElement('img');
+  const cardTitle = document.createElement('h4');
+  const commentAndLikeDiv = document.createElement('div');
+  const commentBtn = document.createElement('button');
+  const pLikesTag = document.createElement('p');
+  const likeHeart = document.createElement('div');
+
+  cardContainer.append(cardElement);
+  cardElement.append(cardImage, cardTitle, commentAndLikeDiv);
+  cardElement.classList.add('item');
+  cardTitle.classList.add('card-title');
+  likeHeart.classList.add('heart');
+  likeHeart.id = item.id;
+  commentAndLikeDiv.append(likeHeart, pLikesTag, commentBtn);
+  pLikesTag.classList.add('p-like');
+  commentAndLikeDiv.classList.add('comment-and-like');
+  commentBtn.classList.add('comment-btn');
+  commentBtn.setAttribute('movie_id', item.id);
+  commentBtn.id = item.id;
+  commentBtn.textContent = 'Comment';
+
+  pLikesTag.textContent = `${itemLike.likes} likes`;
+  cardImage.src = item.image.medium;
+  cardTitle.textContent = item.name;
+};
```

Comment:
- [ ] Your likes do not persist after page reload. As required, when the page is refreshed the number of likes for each item on the homepage, as recorded on the Involvement API, should be displayed. Kindly fix that.

  Items are liked | Page is refreshed
  :---:|:---:
  ![ezekiel-1](https://user-images.githubusercontent.com/36542986/163360429-cf6dbca4-1225-413d-a7eb-98df474313c2.jpg) | ![ezekiel-2](https://user-images.githubusercontent.com/36542986/163360444-d161f805-24b5-4b04-8c29-e2f335c9fac3.jpg)